docs/adr/0004-generate-sboms-with-witness.md

```diff
@@ -0,0 +1,19 @@
+# 4. SBOM Generation with Witness
+
+Date: 2022-03-29
+
+## Status
+
+Accepted
+
+## Context
+
+SBOM are required for software running on government hardware per EO14028.
+
+## Decision
+
+Using Witness' Syft attestor functionality allows Zarf to continue to get more refined SBOM capabilities as Witness' capabilities expand over time. Syft is capable of finding installed packages and some binaries for statically compiled dependencies over each image within a Zarf package. This allows for SBOMs for each image to be generated and packaged along with the Zarf package.  Capabilities to display SBOMs for a Zarf package are forthcoming.
```

Comment:
"apabilities to display SBOMs for a Zarf package are forthcoming.", would love for that to have more info.